core/api/transport/network_address.hpp

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_API_TRANSPORT_NETWORK_ADDRESS_HPP
+#define KAGOME_CORE_API_TRANSPORT_NETWORK_ADDRESS_HPP
+
+#include <cstdint>
```

Comment:
why it is here

Reply:
removed the file. no file - no problem :)